default_gateway.go

```diff
@@ -119,15 +130,15 @@ func (sb *sandbox) needDefaultGW() bool {
 
 func (sb *sandbox) getEndpointInGWNetwork() *endpoint {
 	for _, ep := range sb.getConnectedEndpoints() {
-		if ep.getNetwork().name == libnGWNetwork {
+		if ep.getNetwork().name == libnGWNetwork && strings.HasPrefix(ep.Name(), "gateway_") {
```

Comment:
From this change, it seems it is not possible for windows to create a new nat network with a different name.
Is this correct ?